Various (#59)
eval/envied: README.md

- there are better ways: e.g. a sample file in a project with a bunch of exports (ie `export FOO=sane-default # and even some documentation`) that someone can source in their shell (see 'Best Practices')
- made the code quite complex

## Development

- install [direnv](http://direnv.net/)
- cp .envrc{.sample,}
- direnv allow

```bash
$ setup

# run tests
$ rspec

# hack with pry
$ console

# run CLI:
$ envied
```

## Contributing

> For new feature-suggestions open an issue before opening a PR.

Reviewer: Perhaps this is easier to read?

```suggestion
To suggest a new feature, open an Issue before opening a PR.
```

Author: :+1: thanks!
